### PR TITLE
Instructions for shared port

### PR DESCRIPTION
**Pull request by llobato**
Instructions for FE admins to configure the shared_port configuration for collectors + ccbs.

### CODE REVIEW

**Comment by efajardo** [comment]
@llobato, @brianhlin 

I think I would like it to be written as. If you are upgrading a "standalone" submit host the "default" open port has changed from `9618` to `9615` and you need to upgrade your firewall rules to reflect this change. You can figure out which port would be used by running the following command:

```
condor_config_val SHARED_PORT_ARGS
```

**Comment by llobato** [comment]
It's OK with me the change, but you might want to correct 

>" the "default" open port has changed from `**9618**` to `**9615**` and.."

Port has not changed from `9618` to `9615`. **It was changed from `9615` to `9618`**

File: docs/other/install-gwms-frontend.md [hunk 1]
```diff
@@ -50,11 +50,15 @@ Before starting the installation process, consider the following points (consult
 !!! note
     GlideinWMS versions prior to 3.4.1 also required port 9615 for the Schedd,
     and did not support using shared port for the secondary collectors.
+    If you are upgrading a standalone submit host from version 3.4 or earlier, the default open 
+    port has changed from 9615 to 9618 and you need to upgrade your firewall rules to reflect this change.
```
Comment (matyasselmeci) [inline docs/other/install-gwms-frontend.md]:
```suggestion
    port has changed from 9615 to 9618, and you need to update your firewall rules to reflect this change.
```

File: docs/other/install-gwms-frontend.md [hunk 2]
```diff
@@ -50,11 +50,15 @@ Before starting the installation process, consider the following points (consult
...
+    You figure out which port would be used by running the following command:
```
Comment (matyasselmeci) [inline docs/other/install-gwms-frontend.md]:
```suggestion
    You can figure out which port will be used by running the following command:
```

**Pull request merged**